### ISSUE TITLE
SVG in Markdown is not shown

### ISSUE THREAD

**Issue by NetoBF** [doxygen/doxygen#11233]
**Describe the bug**
I tried to add an local .svg-image (created in draw.io) to a Markdown-page (our main-page, also used as mainpage for doxygen) the following way:

```md
![quick actions](landing_page.svg)
```
(alternatively with a path, that doesn't matter)

But it just won't work. it only shows the alternative name and that's it. 
In the same markdown file I am also referencing a png-image the same way and that works perfectly. 

This is shown in the web-developer-view:
![grafik](https://github.com/user-attachments/assets/344eb065-3227-4af6-8c16-c4a8b6147c40)

Also I noticed it's also not working in our git-azure-view, while the other picture works there as well.
But it does work inside VSCode in the markdown-preview. There the picture is shown correctly. 

The SVG contains links. the idea was to have some quick-link-buttons at our front page. I cannot really attach the example since the links are of course nothing to publish. But I also tried to just create a simple box in draw.io without any links and stored it as svg. and it also won't be shown. So if you try the same it should also be that outcome. 
with the test-box in VSCode:
![grafik](https://github.com/user-attachments/assets/140c51ce-fbc4-47bd-9f8f-82182e49f364)

and in doxygen:
![grafik](https://github.com/user-attachments/assets/79370ed2-bfc0-46aa-9c89-4c0b0ec28b98)

The doxygen-log also does not mention anything.

I am using doxygen version 1.10 (via apt-get) in wsl 2 Ubuntu 22.04

**Comment by doxygen** [comment]
Did you tell doxygen to copy the image file to the HTML output via `HTML_EXTRA_FILES`? 

If so, then please attach a zip or tar with a self-contained example (source+config file).

**Comment by albert-github** [comment]
The solution provided by doxygen (with `HTML_EXTRA_FILES`) should work but is probably a bit quite a bit of work in case of a lot of files. 
I think it probably would be better to add the path of the image file(s) to the `IMAGE_PATH` though this is not stated in the doxygen manual, for this I've just pushed a proposed patch, pull request #11234

Regarding:
> Also I noticed it's also not working in our git-azure-view, while the other picture works there as well.
> But it does work inside VSCode in the markdown-preview. There the picture is shown correctly.

I think this is outside the scope of doxygen but is also a bit unclear here (now), would probably be another issue as well.

>The SVG contains links.

Without an example, as a separate discussion / issue, it is hard to tell what the problem is, but probably your guess that it is because of the not shown image is correct.

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by NetoBF** [comment]
Hello, thank you for your answers.
Because I mentioned it: in azure this is also a bug and there is an open ticket since 2018. So not related to doxygen (I wasn't just sure if I am the problem here).

I now tried adding the ```IMAGE_PATH``` and yes, that was definetly the problem, thank you! The SVG is shown now, but still the embedded links do not work. 

In the meanwhile I found a workaround by splitting it into smaller pictures as pngs and layouting it in md itself but somehow, in contrary to the webview, doxygen puts all images including their captions below each other. the webview of the markdown-file has them side-by-side as expected. But I also had them side by side in doxygen without captions before I added the image-path. Unfortunately I cannot reproduce this somehow now. 
Anyway that's another issue. 

so at least the svg is shown now. Which means this problem here is solved.

**Comment by albert-github** [comment]
Indeed the other things are other, separate issues.
- links, can you show, maybe in a discussion, what you have
- side by side images are possible but you have to be careful with the size of the images especially you need in the line:
  ```
  <svg version="1.1" ...
  ```
  the `width`, `height` and, probably, `viewBox` attributes, like
  ```
  <svg version="1.1" width="104" height="31" viewBox="0 0 104 31"
  ```
  Also be careful with the whitespace and newlines between the "image" statements

**Comment by NetoBF** [comment]
for the links in the svg, that I create with draw.io, I have two possibilities: storing it as draw.io-compatible svg-format or exporting it as an svg. It doesn't matter which option I choose, the image somehow does not allow clicking it to open the url. I made another test with that just linking to github.
maybe it helps showing the generated html:
![grafik](https://github.com/user-attachments/assets/ec4cecb6-08b3-46dd-bd0c-93ae8c38dc11)
As you can see the link exists in the svg. Also doxygen seems to use the "object" instead of the "img" cmd, which is also correct according to some answers online. or is draw.io the problem here?

**Comment by albert-github** [comment]
In principle the image does look OK, when I just use an image with a link inside it it works. It must have to do with the "img" / "iframe" command. Best I sto file another issue for this problem.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
